man/ostree.repo-config.xml

```diff
@@ -122,6 +122,27 @@ Boston, MA 02111-1307, USA.
         keep free. The default value is 3.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>add-remotes-config-dir</varname></term>
```

Comment:
Minor bikeshed...what do you think about `write-remotes-config-dir`?

Reply:
Oh, that would be fine. I meant to add in the PR that I was open to suggestions on the name.